.github/workflows/containerize-invehicle-stack-agemo.yml

```diff
@@ -0,0 +1,51 @@
+# SPDX-License-Identifier: MIT

```

Comment:
The specified license in the LICENSE file in the root directory of this repository is Apache 2.0. Do you really want to license the workflows under another license here? 

In addition, the Eclipse Project handbook proposes to include a full copyright header in addition to the License Identifier. See #16 for more details.

Reply:
I am waiting for some confirmation from our side about the license question

Reply:
@ladatz any updates on the licensing topics here?

Reply:
@eriksven As far as I know, we are still waiting on our open source office team which is reviewing contribution process. I heard we had made some progress and I will update this once I have the green light

Reply:
@eriksven We are good to merge this. I have updated the license to Apache 2, to be consistent with the rest of the blueprint